ruleset.go

```diff
@@ -104,13 +104,14 @@ type ListRulesetRulesResponse struct {
 
 // RuleActions represents a rule action
 type RuleActions struct {
-	Suppress    *RuleActionSuppress     `json:"suppress,omitempty"`
 	Annotate    *RuleActionParameter    `json:"annotate,omitempty"`
-	Severity    *RuleActionParameter    `json:"severity,omitempty"`
-	Priority    *RuleActionParameter    `json:"priority,omitempty"`
-	Route       *RuleActionParameter    `json:"route"`
 	EventAction *RuleActionParameter    `json:"event_action,omitempty"`
 	Extractions []*RuleActionExtraction `json:"extractions,omitempty"`
+	Priority    *RuleActionParameter    `json:"priority,omitempty"`
+	Severity    *RuleActionParameter    `json:"severity,omitempty"`
+	Suppress    *RuleActionSuppress     `json:"suppress,omitempty"`
+	Suspend     *RuleActionSuspend      `json:"suspend,omitempty"`
+	Route       *RuleActionParameter    `json:"route"`
```

Comment:
That's a good question. I thought that this was because of the go format tool I am using here, but now I remember that I ordered it like that to match the alphabetical order we have on the [API schema](https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/paths/~1rulesets~1%7Bid%7D~1rules~1%7Brule_id%7D/get). I am totally fine to rollback this, if necessary 👍

Reply:
No need, was more curious than anything. Thank you for the clarification!